src/box/lua/net_box.c

```diff
@@ -58,6 +59,18 @@
 
 #define cfg luaL_msgpack_default
 
+enum {
+	/**
+	 * IPROTO protocol version supported by the netbox connector.
+	 */
+	NETBOX_IPROTO_VERSION = 0,
+};
+
+/**
+ * IPROTO protocol features supported by the netbox connector.
+ */
+static struct iproto_features NETBOX_IPROTO_FEATURES;
```

Comment:
Why is it in caps? The same for `IPROTO_CURRENT_FEATURES`. Don't we usually use caps for macros and enums? And in Lua for constants.

Reply:
This is to emphasize that they are constants. If I could, I'd make them macros. Plus it's consistent with `IPROTO_CURRENT_VERSION`.